test/disable-user-indexes/mzcompose.yml

```diff
@@ -0,0 +1,148 @@
+
+      - step: kill-services
+        services: [materialized]
+
+  user-indexes-disabled:
+    steps:
+      - step: run
+        service: materialized
+        daemon: true
+        force_service_name: true
+        command: >-
+          --disable-user-indexes
+          --data-directory=/share/mzdata
+          --experimental
+
+      - step: wait-for-mz
+
+      - step: run
+        service: testdrive-svc
+        command:
+           - user-indexes-disabled.td
+
+services:
+  # This should rightly have a dependency on materialized, but that we want to
+  # use it to connect to Materialized with user indexes enabled and disabled
+  # causes friction.
+  testdrive-svc:
+    mzbuild: testdrive
+    entrypoint:
+      - bash
+      - -c
+      - >-
+        testdrive
+        --kafka-addr=kafka:9092
+        --schema-registry-url=http://schema-registry:8081
+        --materialized-url=postgres://materialize@materialized:6875
+        --no-reset
+        --max-errors=1
+        $$*
+      - bash
+    volumes:
+      - .:/workdir
+
+  materialized:
+    mzbuild: materialized
+    command: >-
+      --data-directory=/share/mzdata
+      --experimental
+      --disable-telemetry
+    ports:
+      - 6875
+    environment:
+    - MZ_DEV=1
+    volumes:
+      - mzdata:/share/mzdata
+
+  zookeeper:
+    image: confluentinc/cp-zookeeper:5.5.4
+    environment:
+    - ZOOKEEPER_CLIENT_PORT=2181
+    volumes:
+      - zk-data:/var/lib/zookeeper
+  kafka:
+    image: confluentinc/cp-kafka:5.5.4
+    environment:
+    - KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181
+    - KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka:9092
+    - KAFKA_AUTO_CREATE_TOPICS_ENABLE=false
+    - KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=1
+    - KAFKA_CONFLUENT_SUPPORT_METRICS_ENABLE=false
+    - KAFKA_MIN_INSYNC_REPLICAS=1
+    - KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR=1
+    - KAFKA_TRANSACTION_STATE_LOG_MIN_ISR=1
+    - KAFKA_BROKER_ID=1
+    volumes:
+      - kafka-data:/var/lib/kafka/data
+  connect:
+    image: debezium/connect:1.4
+    environment:
+    - BOOTSTRAP_SERVERS=kafka:9092
+    - GROUP_ID=1
+    - CONFIG_STORAGE_TOPIC=connect_configs
+    - OFFSET_STORAGE_TOPIC=connect_offsets
+    - KEY_CONVERTER=io.confluent.connect.avro.AvroConverter
+    - VALUE_CONVERTER=io.confluent.connect.avro.AvroConverter
+    - CONNECT_KEY_CONVERTER_SCHEMA_REGISTRY_URL=http://schema-registry:8081
+    - CONNECT_VALUE_CONVERTER_SCHEMA_REGISTRY_URL=http://schema-registry:8081
+    depends_on: [kafka, schema-registry]
+  schema-registry:
+    image: confluentinc/cp-schema-registry:5.5.4
+    environment:
+    - SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS=PLAINTEXT://kafka:9092
+    - SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL=zookeeper:2181
+    - SCHEMA_REGISTRY_HOST_NAME=schema-registry
+    - SCHEMA_REGISTRY_LISTENERS=http://schema-registry:8081,http://localhost:8081
+    depends_on: [kafka, zookeeper]
+
+volumes:
+  mzdata:
+  kafka-data:
+  zk-data:
```

Comment:
noooooo not more mzcompose https://github.com/MaterializeInc/materialize/issues/6938

Can you work with @philip-stoev to use his new Python-based workflows to avoid this? If possible I think this should just be a new workflow in the main `test/testdrive` suite.

Reply:
Ah, I was largely unaware of the Python workflows; will work with him on that next week.

Reply:
Offline, Nikhil mentioned that we don't need to block the PR on this, so will merge on next green and follow up on cleaning this up.

Reply:
I can confirm that I did say this.